docs/guides/airflow-quickstart.md

```diff
@@ -30,6 +31,20 @@ Before you begin, make sure you have installed:
 
 > **Note:** We recommend that you have allocated at least **2 CPUs** and **8 GB** of memory to Docker.
 
+## Install the OpenLineage Airflow Provider
+
+Use the following commands to install the official OpenLineage Provider and its requirements:
+
+```bash
+pip install apache-airflow-providers-openlineage && 
+pip install apache-airflow-providers-common-sql && 
+pip install attrs && 
+pip install openlineage-integration-common && 
+pip install openlineage-python
```

Comment:
This is probably too much.
I think
```
apache-airflow-providers-openlineage
apache-airflow-providers-common-sql
```
should go into `requirements.txt` and it should be enough.